conans/test/integration/toolchains/cmake/test_ninja.py

```diff
@@ -140,9 +140,9 @@ def test_locally_build_linux(self):
         self.assertIn("architecture: i386:x86-64", self.client.out)
         self.assertIn("DYNAMIC", self.client.out)
         self.client.run_command("file libfoobard.so")
-        # FIXME: Broken assert
-        #  self.assertIn("with debug_info", self.client.out)
+        self.assertIn("debug", self.client.out)
```

Comment:
The output of this check in Linux was completely unrelated.

Reply:
Do you have an example? The pr-head job didn't show anything, and running on Docker here I can't see that problem. Maybe I need to add a test tag

Reply:
@uilianries  what's the status? do you need some help, e.g. debugging assert on Mac?

Reply:
@SSE4 This error occurs on Linux, but I can't reproduce. If try to run this test, it works for me. Maybe it's related to execution order with other tests.

Reply:
isn't it OSX only test? or are you cross-compiling from linux to OSX?

Reply:
Not this test. The OSX test is at line 182: https://github.com/conan-io/conan/pull/8005/files#diff-fe17138483443cf5c9a968002a35f4132bb14e0b6e7d05ea0079f45e6f9636baR182

Reply:
No cross-compiling too, only building on Linux to Linux.

Reply:
I don't understand. PR title says `Validate CMake Toolchain + Ninja on OSX`. does it also introduces some Linux changes (in such case title has to be updated)? if not, why OSX changes affect on Linux build?

Reply:
Good question, this PR adds a test for OSX, but on my previous PR I added Linux and it doesn't work for the same unknown reason. So, I added here again. I should remove it and investigate better and open a new PR I think

Reply:
yes, I think better not to mix unrelated things here